sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/CertificateIssuer.java

```diff
@@ -145,7 +145,7 @@ public String getOrganizationId() {
      * @param organizationId the org id to set.
      * @return the Issuer object itself.
      */
-    public Issuer setOrganizationId(String organizationId) {
+    public CertificateIssuer setOrganizationId(String organizationId) {
```

Comment:
isssuer -> issuer, [here](https://github.com/Azure/azure-sdk-for-java/pull/6136/files#diff-5d406c82e9fe02340b6bfbe7eb954273R136)